app/tasks/utils.py

```diff
@@ -104,6 +105,20 @@ def get_or_create_workdir():
     return WORKDIR
 
 
+def query_result_to_csv(filename, result):
```

Comment:
Un peu verbeuse la méthode sur l'écriture du header.
Peut-être moyen de faire un truc genre
`outcsv.writerows([row.name for row in result.cursor.description])`
qui replacerait avantageusement tout le for + la décla de variable, etc...

Reply:
remarque prise en compte